src/datasets-cli/index.ts

```diff
@@ -52,6 +52,8 @@ function makeTypeForProperty(args: { property: PropertySchema }): string {
   switch (args.property.type) {
     case PropertyTypesEnum.String:
       return 'string';
+    case PropertyTypesEnum.ListOfStrings:
+      return `(${args.property.defaultValue?.map((value) => `'${value}'`).join(' | ')})[]`;
```

Comment:
this could be values other than the default value
```suggestion
      return `string[]`;
```